docs/theming.md

```diff
@@ -24,10 +24,10 @@ To define a theme for Element:
 In future, the assets for a theme will probably be gathered together into a
 single directory tree.
 
-Custom Themes
-=============
+ JSON Based Themes (Labs Feature):
+==================================
 
-Themes derived from the built in themes may also be defined in settings.
+When [`feature_custom_themes`](https://github.com/vector-im/element-web/blob/master/docs/labs.md#custom-themes-feature_custom_themes) is enabled, JSON based themes derived from the built in themes may also be defined in settings.
```

Comment:
This isn't true though, you don't need the Lab enabled to have a custom theme in config.json - the labs flag lets the user add such a custom theme by a JSON URL.

Reply:
@t3chguy thanks for reviewing! what further modifications do you suggest in order to make this pr a valid one?.

Reply:
GitHub won't let me suggest on this line but I guess you could change it to something like

```
JSON themes derived from the built in themes may be defined in the `config.json` file. When [`feature_custom_themes`](https://github.com/vector-im/element-web/blob/master/docs/labs.md#custom-themes-feature_custom_themes) is enabled, users can add new JSON themes in the Appearance section of Settings.
```